doc/processes/release_process.md

```diff
@@ -0,0 +1,109 @@
+or the Apache License, Version 2.0 which accompanies this distribution and
+is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+This Source Code may also be made available under the following
+Secondary Licenses when the conditions for such availability set
+forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+General Public License, version 2 with the GNU Classpath
+Exception [1] and GNU General Public License, version 2 with the
+OpenJDK Assembly Exception [2].
+
+[1] https://www.gnu.org/software/classpath/license.html
+[2] http://openjdk.java.net/legal/assembly-exception.html
+
+SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# Eclipse OpenJ9 Release Process
+
+The OpenJ9 JVM is composed of two code repositories:
+
+* OpenJ9 [https://github.com/eclipse/openj9]()
+* OMR [https://github.com/eclipse/openj9-omr]()
+
+OpenJ9 needs to be combined with code from OpenJDK to create a Java SDK.
+The OpenJDK code is managed in separate github repos referred to as the 
+"OpenJDK extensions for OpenJ9 for JDK X".  Currently there are two 
+extensions repos:
+
+* JDK8 [https://github.com/ibmruntimes/openj9-openjdk-jdk8]()
+* JDK9 [https://github.com/ibmruntimes/openj9-openjdk-jdk9]()
+
+OpenJ9 binaries are built by the AdoptOpenJDK community.  
+
+An OpenJ9 release should:
+
+* Use a single code base to support the various JDK-levels.
+* Track the upstream OpenJDK releases.  
+* Use a consistent set of tags across OpenJ9 & extensions repos to 
+ensure that releases can be rebuilt or branched later.
+* OpenJ9 releases should be high quality and pass a common quality bar. 
+* Communicate the tags for a release to AdoptOpenJDK to ensure that
+binaries at the correct levels have been created. 
+* Clearly define the supported platforms
+* Should not regress performance from release to release.
+
+
+## Release cadence
+OpenJ9 should aim to release quarterly to match the OpenJDK long term 
+support (LTS) release cadence.  OpenJDK 8 is released quarterly and 
+other LTS releases will likely follow similar release cadences.  There 
+should be monthly milestone builds as the project builds towards the 
+next release.
+
+## Quality bar
+Work with the AdoptOpenJDK community to define a common quality bar for 
+any OpenJDK-based JDK.  The aim is to define a set of tests, both 
+functional and stress, and 3rd party applications that should pass for 
+every release.
+
+## Platform support
+Define the platforms that are supported by this release.  The set of 
+platforms will depend on the availability of hardware at OpenJ9 and 
+AdoptOpenJDK that can be used to validate the platform meets the quality 
+bar.  The code base may support more platforms than can be validated by 
+the quality.  Such platforms will not be officially included in the release.
+
+## Performance
+OpenJ9 should avoid regressing performance from release to release.
```

Comment:
This is a good goal to have. There are times where a regression may not be identified late in the process or that the project has taken a decision to pay some performance for a new feature/capability. We need a way to document these decisions and account for them in testing. Sometimes things get slower before they get faster.

We also need to keep in mind performance is a multi-aspect thing which can have competing demands - the project may choose to trade start-up for throughput, throughput for start-up, and then there are the myriad of memory usage and compile-time metrics.

At the very least we need to have tolerances for noise and other such things. Probably an area we need to flesh out in more details before we can say we have a fully defined 'plan'.

Reply:
Thanks for the feedback.   I agree parts of this are aspirational and that there may be times when a regression is necessary - either late discovery of the cause or due to necessary bug fix.

Starting from a goal of "no performance regressions", each release should document a set of measures that can the project will track and ensure don't regress.  If the measures do regress, then the reason should be called out in release issue.

Sound reasonable?

Reply:
I think that sounds fine though I think 'no performance regressions' does need to include a noise margin - variations in benchmarks mean that on some benchmarks regressions less than a certain % are not considered significant. Given the difficulty of running perf testing as well as finding and fixing issues found from perf testing having the wording here such that we admit and accept (as policy) that there are tolerances for regressions and that those tolerances are often non-zero. Setting tolerances etc is a test specific thing, but I think codifying that we expect them to exist and that setting them is part of tests being added to release criteria and that they are subject to periodic review would seem the most prudent course to me based on past experience with perf measurement.

Reply:
I like the revisions to this section - my concerns have been addressed - thank!